### PR TITLE
Renamed from couch_tap to couchdb_to_sql

### LINKED ISSUE #16: Rename gem to couchdb_to_sql
Since our gem has significant changes from the original `couch_tap` implementation, we (me and @afinne) discussed and decided to rename it to `couchdb_to_sql` - a name which clearly tells what it does, and also clearly distinguishes it from @samlown's great original work, from which we are derived.

The module name will be `CouchdbToSql`, in line with [Rubygems: Name your Gem](http://guides.rubygems.org/name-your-gem/)

### PR DESCRIPTION
**Pull request by perlun**
This makes the projects easier to distinguish between, and we will also publish the gem now to Rubygems.org, which makes more sense since the project is open source.

Closes #16.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,52 @@
...
 
 Utility to listen to a CouchDB changes feed and automatically insert, update,
-or delete rows into a relational database from matching key-value conditions of incoming documents.
+or delete rows into an SQL database from matching key-value conditions of incoming documents.
 
-While CouchDB is awesome, business people probably won't be
-quite as impressed when they want to play around with the data. Regular SQL
-is generally accepted as being easy to use and much more widely supported by a larger
-range of commercial tools.
+`couchdb_to_sql` is heavily indebted to [samlown's](https://github.com/samlown) original [couch_tap](https://github.com/samlown/couch_tap) gem, which is today unfortunately quite abandoned. We have added functionality needed for our particular use case, while still trying to keep it reasonably flexible and not too hardwired to the `ember-pouch` use case.
```
Comment (olleolleolle) [inline README.md]: Suggestion: `which is today unfortunately quite abandoned` can be dropped.

File: README.md [hunk 2]
```diff
@@ -1,59 +1,52 @@
...
 
-Couch Tap will listen to incoming documents on a CouchDB's changes
-stream and automatically update rows of RDBM's tables defined in the
-conversion schema. The changes stream uses a sequence number allowing
-synchronization to be started and stopped at will.
+While CouchDB is awesome, business people probably won't be quite as impressed when they want to play around with the data. Regular SQL is generally accepted as being easy to use and much more widely supported by a larger range of commercial tools.
 
-Ruby's fast and simple [sequel](http://sequel.jeremyevans.net/) library is used to provide the connection to the
-database. This library can also be used for migrations, important for frequently changing schemas.
+`couchdb_to_sql` will listen to incoming documents on a CouchDB's changes stream and automatically update rows of the SQL database tables defined in the conversion schema. The changes stream uses a sequence number allowing synchronization to be started and stopped at will.
```
Comment (olleolleolle) [inline README.md]: Suggestion: changes stream => [Changes feed](http://docs.couchdb.org/en/2.1.0/api/database/changes.html?highlight=changes#changes-feeds)

Makes me wonder: which kind of Changes Feed is it? A continuous one?

File: README.md [hunk 3]
```diff
@@ -1,59 +1,52 @@
...
 
-Couch Tap takes a simple two-step approach converting documents to rows. When a change event is received
-for a matching `document` definition, each associated row is completely deleted. If the change
-is anything other than a delete event, the rows will be re-created with the new data.
-This makes things much easier when trying to deal with multi-level documents (i.e. documents of documents)
-and one-to-many table relationships.
+Ruby's fast and simple [sequel](http://sequel.jeremyevans.net/) library is used to provide the connection to the database. This library can also be used for migrations, which is important for frequently changing schemas.
```
Comment (olleolleolle) [inline README.md]: Suggestion: `Ruby's fast and simple [sequel](http://sequel.jeremyevans.net/) library` => `[Sequel](http://sequel.jeremyevans.net/)`

**Review by olleolleolle — APPROVED** [review]
Make the change! 🎉

File: README.md (hunk 3 above)
Comment (perlun) [reply README.md]: Thanks, that wording was inherited from the original version of the document.

File: README.md (hunk 2 above)
Comment (perlun) [reply README.md]: The continous one, yes: https://github.com/ecraft/couchdb_to_sql/blob/f9504051bcb76a14afa5e4d92a720368a6438a19/lib/couch_tap/changes.rb#L117-L122

**Pull request merged**